frontend/src/sass/styles.scss

```diff
@@ -49,6 +49,14 @@ a {
 	height: auto;
 }
 
+.file-link {
+	display: none;
+	visibility: hidden;
```

Comment:
`display: none;` and `visibility: hidden;` should not be compatible with each other.